src/components/Pie/Pie.tsx

```diff
@@ -12,13 +12,16 @@ type Props = {
 	loading?: boolean;
 	donut?: boolean;
 	labels?: boolean;
+	loadingClassName?: string;
+	emptyClassName?: string;
+	segmentClassName?: string;
```

Comment:
Remove these props please.

All themeing will be done via `className` applied to root element, hardcoded BEM names for all children.

If people want to apply a className just for loading states we will apply the "className" prop to all root elements that could be rendered.

They can then do this in the consumer

```
//before
<Pie loading={loading} loadingClassName={"pie_loading"}/>
```
```
// after
<Pie loading={loading} className={cx(loading && styles.pieLoading))}
```
Both ways achieve same effect with no additional props.

Reply:
Done! Thanks!